### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by Zeus386** [aiaemx/disboard-multiserver-bumper#1]
Exception ignored in thread started by: <bound method GatewayServer._response_loop of <discum.gateway.gateway.GatewayServer object at 0x7fb20485d890>>
Traceback (most recent call last):
  File "/home/runner/BOF/venv/lib/python3.8/site-packages/discum/gateway/gateway.py", line 300, in _response_loop
    function(resp, **params)
  File "main.py", line 106, in bump
    bot.triggerSlashCommand(botID, channelID=channelID, guildID=guildID, data=data, sessionID=bot.gateway.session_id)
TypeError: triggerSlashCommand() got an unexpected keyword argument 'sessionID'

**Comment by aiaemx** [comment]
Did you check if you put in the server and channel ids correctly?

**Comment by Zeus386** [comment]
> Did you check if you put in the server and channel ids correctly?

Yes, I did everything correctly, in fact, the first time I started the bot it worked
This is my code: [replit](https://replit.com/@Zeus386/Discord-Auto-Bumper#main.py)

**Comment by aiaemx** [comment]
i dont really get what the issue is. if you want me to look at it further, add me on discord cherryy#0001
